app/main.go

```diff
@@ -2,14 +2,62 @@ package main
 
 import (
 	"fmt"
-	"github.com/ministryofjustice/opg-modernising-lpa/internal/fake"
+	"github.com/ministryofjustice/opg-go-common/env"
+	"html/template"
+	"log"
+	"net/http"
 )
 
 func Hello() string {
 	return "Hello, world!"
 }
 
+type PageData struct {
+	WebDir      string
+	Prefix      string
```

Comment:
since this will be always be hosted at the root (right?) we shouldn't need a prefix (and I've kind of hated how it has spread everywhere)

Reply:
Cool - I was struggling to see how this worked in `opg-sirius-lpa-frontend` where I pinched it from so I'm more than happy to drop it out.